tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -968,4 +970,31 @@ describe(__filename, () => {
       );
     });
   });
+
+  it('does not render anything when an API error is thrown', () => {
```

Comment:
Why is this needed? It doesn't seem necessary to solve the bug of rendering a 404 page.